_episodes/01-introduction.md

```diff
@@ -1,15 +1,116 @@
+-   Know the different components of ESMValTool
+-   Understand the concept of a community-driven software project
+
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+-   ESMValTool provides a reliable interface to analyse and evaluate climate data
+-   Using ESMValTool promotes standardization, collaboration, and reuse
+-   ESMValTool is built and maintained by an active community of scientists and developers
+-   ESMValTool is written in Python, but supports diagnostic scripts in multiple languages
+
 ---
-FIXME
 
-{% include links.md %}
+## What is ESMValTool
+
+EMSValTool is first and foremost a tool to analyse climate data. But you probably already knew that and we like to think there's more to it than that. So let's start with a quick check to synchronize our expectations.
+
+> ## Question: what is ESMValTool
+>
+> Which of the following items would you say apply to ESMValTool?
+>
+> - A tool to analyse climate data
+> - The easy way out
+> - A community effort
+> - Free
+> - A command line tool
+> - A way to make climate science more [FAIR](https://fair-software.eu/about)
+> - Perfect
+> - Suitable for (Jupyter) notebooks
+>
+> Check our answers by unfolding the box below.
+>
+> > ## ESMValTool is
+> >
+> > &#10003; **A tool to analyse climate data.**  It takes care of finding, opening, checking, fixing, concatenating, and preprocessing CMIP data and several other supported datasets.
+> >
+> > &#10005;  **The easy way out.** If you just want to do an exploratory analysis or quickly hack something together, ESMValTool is probably not the way to go. The tool is intended for robust, repeatable and shareable climate analysis. That *does* require a bit more effort.
+> >
+> > &#10003; **A community effort.** EMSValTool is developed and maintained by a large team of climate scientists and software engineers. It is an open source project to which anyone can contribute. Its longevity depends on these contributions.
+> >
+> > &#10003; **Free.** ESMValTool is licenced under Apache 2.0, which means everyone can use, modify, or share it free of charge. However, we *do* encourage all users to contribute to the community once they get more comfortable with the tool.
+> >
+> > &#10003; **A command line tool.** ESMValTool was originally designed for the command line. But, we are working on a user-friendly python interface as well.
+> >
+> > &#10003; **A way to make climate science more [FAIR](https://fair-software.eu/about).** ESMValTool collects provenance information about the data and code that are used to obtain a result. It comes with a readable recipe format that makes climate analysis consistent, reproducible, and easy to share.
+> >
+> > &#10005;  **Perfect.** Although we are continuously working to improve the tool, you may encounter some bugs or missing features. In this lesson, you will learn how to troubleshoot, find help, and maybe even contribute to the solution yourself.
```

Comment:
I am not sure that everybody knows that a "lesson" consists of several episodes. Maybe we could rephrase the second sentence to "In the following episodes, you will learn..." to make sure that nobody expects to understand troubleshooting etc. after the first episode.